Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG FLOW_PHP_VERSION=8.2.11
+ARG FLOW_BASE_IMAGE_TAG_SUFFIX=cli-alpine3.18
+ARG FLOW_BASE_IMAGE_TAG=${FLOW_PHP_VERSION}-${FLOW_BASE_IMAGE_TAG_SUFFIX}
+ARG FLOW_BASE_IMAGE=php:${FLOW_BASE_IMAGE_TAG}
+
+FROM ${FLOW_BASE_IMAGE}
+
+ARG FLOW_VERSION=0.3.3
+
+LABEL maintainer="Norbert Orzechowicz <contact@norbert.tech>"
+
+# Update package list and install dependencies
+RUN apk update && apk add --no-cache \
+    $PHPIZE_DEPS \
+    gmp-dev \
+    git
+
+# Install PHP extensions
+RUN docker-php-ext-install bcmath gmp
```

Comment:
Don't we need also dom? Or it's there already? Also for snappy it may need zip and/or zstd, no?

Reply:
Those are all extensions available out of the box (except those 3 that I installed manually): 

```console
#12 [ 8/10] RUN php -m
#12 0.038 [PHP Modules]
#12 0.038 bcmath
#12 0.038 Core
#12 0.038 ctype
#12 0.038 curl
#12 0.038 date
#12 0.038 dom
#12 0.038 fileinfo
#12 0.038 filter
#12 0.038 ftp
#12 0.038 gmp
#12 0.038 hash
#12 0.038 iconv
#12 0.038 json
#12 0.038 libxml
#12 0.038 mbstring
#12 0.038 mysqlnd
#12 0.038 openssl
#12 0.038 pcre
#12 0.038 PDO
#12 0.038 pdo_sqlite
#12 0.038 Phar
#12 0.038 posix
#12 0.038 random
#12 0.038 readline
#12 0.038 Reflection
#12 0.038 session
#12 0.038 SimpleXML
#12 0.038 snappy
#12 0.038 sodium
#12 0.038 SPL
#12 0.038 sqlite3
#12 0.038 standard
#12 0.038 tokenizer
#12 0.038 xml
#12 0.038 xmlreader
#12 0.038 xmlwriter
#12 0.038 zlib
#12 0.038 
#12 0.038 [Zend Modules]
#12 0.038 
#12 DONE 0.0s

```